### PR TITLE
Complete Math Magician Website

### PR DESCRIPTION
**Pull request by deyemiobaa**
### Set up a complete app with routes

- Add home and quotes page components
- Style components
- Connect components using routers

### CODE REVIEW

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @deyemiobaa,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Nice PR 👌.
- The Linters are passing 👌.

## Required Changes ♻️
_Check the comments under the review._

- [Nice job mate on the full-math-magicians however, the app is missing some basic color for the `key` to show them, and also the `nav-bar` is scrolling down because the calculator is too big you can make the width and the height adjusted for the calculator to give it a more suitable view.

CPT2204261031-500x247


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by deyemiobaa** [comment]
Oh dang! I added a CSS style that overrode the previous ones, totally forget to check the whole thing before pushing 🤦🏾‍♂️

File: src/App.css [hunk 1]
```diff
@@ -71,13 +104,15 @@ ul {
 
 .display {
   width: 100%;
-  height: 70px;
+  height: 15%;
   background: var(--text-area);
 }
```
Comment (t-yanick) [inline src/App.css:110]: - [OPTIONAL] The display of your calculator is not proportionate (it does not match) with the width of your calculator, it goes out of proportion as you resize the page, please kindly fix the issue, as it is a needed skill for a front end developer to stick to the design as much as possible.

**Review by t-yanick — APPROVED** [review]
Hi @deyemiobaa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**